src/openrct2-ui/windows/GuestList.cpp

```diff
@@ -549,35 +570,21 @@ static void window_guest_list_scrollgetsize(rct_window* w, int32_t scrollIndex,
  */
 static void window_guest_list_scrollmousedown(rct_window* w, int32_t scrollIndex, const ScreenCoordsXY& screenCoords)
 {
-    int32_t i, spriteIndex;
-    Peep* peep;
+    int32_t i;
 
     switch (_window_guest_list_selected_tab)
     {
         case PAGE_INDIVIDUAL:
             i = screenCoords.y / SCROLLABLE_ROW_HEIGHT;
```

Comment:
Declaration of `i` can be moved here.

Reply:
its used by both cases of the switch so I think I'll just leave it. I'll init it to 0 though.